cirq-google/cirq_google/engine/engine.py

```diff
@@ -634,7 +634,6 @@ def _serialize_program(
     ) -> any_pb2.Any:
         if not isinstance(program, cirq.AbstractCircuit):
             raise TypeError(f'Unrecognized program type: {type(program)}')
-        program.device.validate_circuit(program)
```

Comment:
Engine no longer validates inside of serialization. As we know this already wasn't doing a great job of catching all the breaks that might come over through engine. Would probably make more sense to couple this up with your new work on the engine emulator. What are your thoughts here @dstrain115 ?

Reply:
Added to the list thx.